stock_move_change_source_location/tests/test_stock_move_change_source_location.py

```diff
@@ -66,33 +71,23 @@ def setUpClass(cls):
                             "product_id": cls.product2.product_variant_ids.id,
                             "product_uom_qty": 60.0,
                             "product_uom": cls.product.uom_id.id,
+                            "location_id": cls.warehouse.lot_stock_id.id,
+                            "location_dest_id": cls.warehouse.wh_output_stock_loc_id.id,
                         },
                     ),
                 ],
             }
         )
 
     def qty_on_hand(self, product):
-        wiz = self.env["stock.inventory"].create(
+        self.env["stock.quant"].create(
```

Comment:
Better to add 'inventory_mode=True' in context.

See: https://github.com/odoo/odoo/blob/15.0/addons/stock/models/stock_quant.py#L177

Reply:
Done.

Reply:
With this change, the tests are broken!

Reply:
Solved.

Reply:
Mmmh, one of the field is not in the allowed ones. 

https://github.com/odoo/odoo/blob/6c7385c74fafbe84042261e29565a77e2ca0c03c/addons/stock/models/stock_quant.py#L803

Weird

Reply:
Ok, you've got it